low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -0,0 +1,116 @@
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from collections import namedtuple
+from framework.base.sspl_constants import DEFAULT_RECOMMENDATION
+
+Alert = namedtuple('AlertInfo',
+                   ['type', 'severity', 'description',
+                    'impact', 'recommendation'])
+
+alert_for_event = {
+    "Fan": {
+        **dict.fromkeys(["Fully Redundant", "Fully Redundant ()"], {
+            "Asserted": Alert(
+                "fault_resolved", "informational",
+                "'{} Sensor' has recovered from a failure. [{}]",
+                "None", "None"),
+            "Deasserted": Alert(
+                "fault", "critical",
+                "'{} Sensor', is Fully redundant . [{}]",
+                "Fan efficiency will be impacted",
+                DEFAULT_RECOMMENDATION)
+        }),
+        **dict.fromkeys(["Lower Non-Recoverable", "Lower Non-Recoverable ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "Lower Non-Recoverable threshold breached for '{}'. [{}]",
+                "Fan speed is outside acceptable limits",
+                DEFAULT_RECOMMENDATION
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "'{} Sensor has recovered from the failure'. [{}]",
+                "None", "None"
+            )
+        }),
+        **dict.fromkeys(["Lower Critical", "Lower Critical ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "Lower Critical threshold breached for '{}'. [{}]",
+                "Fan speed is outside acceptable limits",
+                DEFAULT_RECOMMENDATION
+            ),
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "'{} Sensor' recovered from the failure. [{}]",
+                "None", "None"
+            )
+        }),
+        **dict.fromkeys(["Lower Non-Critical", "Lower Non-Critical ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "Lower Non-Critical threshold breached for '{}'. [{}]",
+                "Fan speed is outside acceptable limits",
```

Comment:
"Fan speed is below expected limits",

Reply:
> "Fan speed is below expected limits",

Fan speed can be below/above expected limit based on threshold.
Keeping it as -

> Upper Non-Recoverable threshold breached for '{}'. [{}]"
   "Its outside expected limits"